content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx

```diff
@@ -679,7 +679,8 @@ This section describes how to build your own optimistic concurrency control. See
 
 - If you use version 4.4.0 or earlier, you cannot use optimistic concurrency control on `update` operations, because you cannot filter on non-unique fields. The `version` field you need to use with optimistic concurrency control is a non-unique field.
 
-- From version 4.5.0, you can filter on non-unique fields in `update` operations, so you can use optimistic concurrency control with `update` operations. To do so, use the [extendedWhereUnique preview feature](/reference/api-reference/prisma-client-reference#filter-on-non-unique-fields-with-userwhereuniqueinput).
+
+Since version 5.0.0 you are able to [filter on non-unique fields in `update` operations](/reference/api-reference/prisma-client-reference#filter-on-non-unique-fields-with-userwhereuniqueinput) so that optimistic concurrency control is being used. You can also enable this feature using the Preview flag `extendedWhereUnique` from versions 4.5.0 to 4.16.2.
```

Comment:
```suggestion
- Since version 5.0.0 you are able to [filter on non-unique fields in `update` operations](/reference/api-reference/prisma-client-reference#filter-on-non-unique-fields-with-userwhereuniqueinput) so that optimistic concurrency control is being used. You can also enable this feature using the Preview flag `extendedWhereUnique` from versions 4.5.0 to 4.16.2.
```